### PR TITLE
add parameter v

### LINKED ISSUE #4: Parameter -v

### PR DESCRIPTION
**Pull request by justinmueller05082000**
fixes #4

### CODE REVIEW

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
```bash
go run . -v -e a <<EOF
a
a
EOF
```

... doesn't work as expected.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Any way to shorten the changes?

**Comment by justinmueller05082000** [comment]
well I only added two if's and a parameter so i dont think so

**Comment by Al2Klimov** [comment]
You've already computed whether a line matches. -v is just about **whether to print** on match or no match. Still no idea?

**Comment by justinmueller05082000** [comment]
Not really

File: main.go [hunk 1]
```diff
@@ -51,6 +52,7 @@ func main() {
 	}
 
 	for {
+		equalComparator = false
```
Comment (Al2Klimov) [inline main.go:55]: Now you're resetting this at the right place, but twice.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [x] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD master)..HEAD^)`
* [x] `git commit --amend --no-edit`
* [x] `git push -f origin feature/add-parameter-v-4`

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Write a test protocol. Also ensure that not specifying -v still works.

**Comment by justinmueller05082000** [comment]
**Test protocol:**

**toGrep.txt:**

```
fo*bar dog cat cat foobar dog cat mouse mouse
tiger dog cat dog mouse mouse zebra elephant
dog cat foobar mouse ze*bra zebra cat dog
tiger dog cat mouse dog dog cat zebra zebra tiger
elephant foobar foobar elephant foobar
```

**Go code:**

**Terminal:** `go run . -v -e  'fo*' < toGrep.txt`

**Output:**

```
tiger dog cat dog mouse mouse zebra elephant
tiger dog cat mouse dog dog cat zebra zebra tiger
```

**Without -v:**

**Terminal:** `go run . -e  'fo*' < toGrep.txt`

**Output:**

```
fo*bar dog cat cat foobar dog cat mouse mouse
dog cat foobar mouse ze*bra zebra cat dog
elephant foobar foobar elephant foobar
```

**Grep command:**

**Terminal:**  `grep -v -e  'fo*' < toGrep.txt`

**Output:**

```
tiger dog cat dog mouse mouse zebra elephant
tiger dog cat mouse dog dog cat zebra zebra tiger
```

**Conclusion:**
Both results are equal except for the colored highlighting in the terminal. The program does still work without using the -v parameter.

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**